interface/forms/misc_billing_options/new.php

```diff
@@ -34,110 +34,220 @@
-            <?php $datetimepicker_showseconds = false; ?>
-            <?php $datetimepicker_formatInput = false; ?>
-            <?php require($GLOBALS['srcdir'] . '/js/xl/jquery-datetimepicker-2-5-4.js.php'); ?>
-            <?php // can add any additional javascript settings to datetimepicker here;
-                // need to prepend first setting with a comma ?>
-        });
-      });
+<head>
+<?php Header::setupHeader(['datetime-picker']);?>
+<title><?php echo xlt('Miscellaneous Billing Options for HCFA-1500'); ?></title>
+<style>
+    @media only screen and (max-width: 768px) {
+        [class*="col-"] {
+        width: 100%;
+        text-align:left!Important;
+    }
+</style>
+<?php
+if ($GLOBALS['enable_help'] == 1) {
+    $help_icon = '<a class="pull-right oe-help-redirect" data-target="#myModal" data-toggle="modal" href="#" id="help-href" name="help-href" style="color:#676666" title="' . xla("Click to view Help") . '"><i class="fa fa-question-circle" aria-hidden="true"></i></a>';
+} elseif ($GLOBALS['enable_help'] == 2) {
+    $help_icon = '<a class="pull-right oe-help-redirect" data-target="#myModal" data-toggle="modal" href="#" id="help-href" name="help-href" style="color:#DCD6D0 !Important" title="' . xla("To enable help - Go to  Administration > Globals > Features > Enable Help Modal") . '"><i class="fa fa-question-circle" aria-hidden="true"></i></a>';
+} elseif ($GLOBALS['enable_help'] == 0) {
+     $help_icon = '';
+}
+
+?>
+</head>
+<body class="body_top">
+    <div class="container">
+       <div class="row">
+        
+            <div class="page-header clearfix">
+               <h2 id="header_title" class="clearfix"><span id='header_text'><?php echo xlt('Misc Billing Options for HCFA-1500'); ?></span><?php echo $help_icon; ?></h2>
+            </div>
+
+        </div>
+        <div class="row">
+            <form method=post <?php echo "name='my_form' " .  "action='$rootdir/forms/misc_billing_options/save.php?id=" . attr($formid) . "'\n";?>>
+                <fieldset>
+                    <legend><?php echo xlt('Select Options for Current Encounter')?></legend>
+                    <div class='col-sm-11 col-offset-sm-1'>
+                        <span class="text"><?php echo xlt('Checked box = yes ,  empty = no');?><br><br></span>
+                        <div class="form-group">
+                            <label><?php echo xlt('Box 10 A. Employment related '); ?>:
+                                <input type="checkbox" name="employment_related" id="box10a" value="1"
+                                <?php
+                                if ($obj['employment_related'] == "1") {
+                                    echo "checked";
+                                }?>>
+                            </label>
+                        </div>
+                        <div class="form-group">
+                            <label><?php echo xlt('Box 10 B. Auto Accident '); ?>:
```

Comment:
Remove the trailing space from within this constant

Reply:
Done